temporal-sdk/src/main/java/io/temporal/internal/common/InternalUtils.java

```diff
@@ -93,10 +90,10 @@ public static Object getValueOrDefault(Object value, Class<?> valueClass) {
 
   public static SearchAttributes convertMapToSearchAttributes(
```

Comment:
Why is this method not moved to SearchAttributesUtil? Looks like it belongs there.